.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
Please remove this file. Also remove the .vscode folder. you can add them to gitignore